### DISCUSSION TITLE
Any plans for a new release?

### DISCUSSION THREAD

**Discussion by igor-baiborodine** [Lednerb/bilberry-hugo-theme#293] [Q&A] [answered]
Hi,

Last release was in October last year. Are you planning a new release? 

Thanks,

Igor Baiborodine

**Comment by Lednerb** [comment]
Hi Igor,

Actually, I would like to publish a new release every time I see the repository, but afterwards I don't have the time for it.

There are some issues and feature requests that I also like to address, but I don't think that I'm able to maintain this theme at the moment and in the near future with the care that is needed to provide a first class user experience.

If you are interested in helping with the project or want to take the role of the official maintainer, I will give you the needed permissions.

In the past you've provided great additions with always good code quality.
Furthermore you're using this theme in production for your own blog so you have deeper insides than myself because although I've created this theme for my own blog, there was never the time for getting started with it.

**Comment by igor-baiborodine** [comment, marked as answer]
Hi Sascha,

I will be happy to help you with the project as an official co-maintainer. You can count on me.

Thanks,

Igor

**Reply by Lednerb** [reply]
Welcome on Board!
